### DISCUSSION TITLE
403 logging into production but not on local

### DISCUSSION THREAD

**Discussion by scvn1** [filamentphp/filament#6819] [Help]
I just deployed the project to production for the first time and I'm getting a 403 after logging in with my new `artisan filament:user` I created.

I'm not using Filament Shield.

Why might this be happening on production but not local?

**Comment by pboivin** [comment]
Have you seen this note in the Users section?
https://filamentphp.com/docs/2.x/admin/users#authorizing-access-to-the-admin-panel

**Reply by scvn1** [reply]
I had email verification line still in there but no "MustVerifyEmail" requirement. Removed and all good.
